content/blog/jsonvalidate-1.1.0.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "jsonvalidate 1.1.0"
+date: 2019-06-25T07:38:50+01:00
+draft: true
+---
+
+[JSON](https://www.w3schools.com/js/js_json_intro.asp) is useful as a data-interchange format, due to the massive popularity of javascript recently.  It's objectively not the best format for anything, being verbose (but not as bad as XML), awkward to write, and lacking in rich types.  However, basically every language supports reading and writing JSON now, so it makes a nice lowest common denominator.
```

Comment:
I'd drop "recently" in the first sentence.

Didn't realise how much you hate json :laughing: a lot of people find it easy to read and write! I'd drop the sentence dissing it and just stick with "Basically every language supports reading and writing JSON now, so it makes a nice lowest common denominator."

Reply:
The lack of comments drives me batty! There's lots of fun things coming out now that generate json (like https://jsonnet.org/) that might make json more fun to write.  But writing the [odin schema](https://github.com/mrc-ide/odin/blob/master/inst/schema.json) stretched my patience.

I don't _hate_ json, and I'll tone it down as it wasn't meant to come out too negative.  At least it's not XML!